rules/opentitan/exec_env.bzl

```diff
@@ -354,4 +367,22 @@ def common_test_setup(ctx, exec_env, firmware):
     for attr, name in ctx.attr.binaries.items():
         update_file_attr(name, attr, exec_env.provider, data_files, param, action_param)
 
+    if ctx.attr.need_jtag:
+        openocd = exec_env.openocd
+        jtag_data_files = [openocd]
```

Comment:
I would suggest to rename `jtag_data_files` to `jtag_data` or `jtag_data_targets` because it's a list of targets, not a list of files.